test-cases/tests/dashboards/e09-verify-grafana-ratelimiting-dashboard.md

```diff
@@ -0,0 +1,28 @@
+---
+products:
+  - name: rhoam
+    environments:
+      - osd-fresh-install
+estimate: 15m
+tags:
+  - per-release
+---
+
+# E09 - Verify Grafana rate-limiting dashboard
+
+## Prerequisites
+
+1. Login via `oc` as a user with **cluster-admin** role (kubeadmin):
+
+```
+oc login --token=<TOKEN> --server=https://api.<CLUSTER_NAME>.s1.devshift.org:6443
+```
+
+2. RHOAM is installed
+3. SSO IDP configured on a cluster
+
+## Steps
+
+1. Go to redhat-managed-api-customer-monitoring-operator namespace
+2. Go to Routes -> grafana-route and login as customer-admin user using testing-idp
+3. Verify the rate-limiting dashboard is present (TODO: more detailed step)
```

Comment:
Perhaps @laurafitzgerald can provide more information on these steps if the Grafana work is added before v0.2.0 is cut

Reply:
Just closing this one as this test case has been automated I believe this'll be updated.

Reply:
updated here in this commit: https://github.com/psturc/integreatly-operator/pull/2/commits/5f1a0f41e2fd4443ee598499c89bb0f7a52931fa